### ISSUE TITLE
In some cases, NVDA does not read graphics in Chromium-based browsers

### ISSUE THREAD

**Issue by Marc2p** [nvaccess/nvda#16973]
When accessing a page, NVDA does not read the graphics present on the web in browse mode.
They are accessible with the object navigator.
If NVDA is restarted (with the page open) the graphics are read correctly, but if the web page is reloaded, the graphics disappear.
### Steps to reproduce:
1) Enter this website with a Chromium-based browser (in my case, Google Chrome or Microsoft Edge):
https://ilumina.mx/juegos/index.html
2) Check that not all the graphics present are read. It may be necessary to reload the page.
3) Restart NVDA (with the web open) the graphics should appear.
4) Reload the page. Graphics do not appear in browse mode
### Actual behavior:

Graphics do not appear in browse mode, unless you restart NVDA with the page open or refresh the content with NVDA+f5
possibly related to this issue: https://github.com/nvaccess/nvda/issues/9500#issue-434512446
### Expected behavior:

Graphics should be visible in browse mode without needing to restart NVDA or refresh content
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2024.2 (2024.2.0.32555)
#### Windows version:
Windows 11 23H2 (AMD64) compilación 22631.3958
#### Name and version of other software in use when reproducing the issue:
Microsoft Edge
Versión 127.0.2651.86 (Compilación oficial) (64 bits)
Google Chrome
Versión 127.0.6533.100 (Build oficial) (64 bits)
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
This is not reproducible anymore in Chromium canary 144.0.7545.0  with NVDA alpha-53511,1472d4b9 (2026.1.0.53511)
Closing as works for me. If you are still having this issue, please comment and we can reopen.
